tests/macros/utils.js

```diff
@@ -4,6 +4,8 @@
 
 // Provides utilities that as a whole constitute the macro test framework.
 
+const fs = require('fs');
+const path = require('path');
```

Comment:
nit: I like putting a blank line between different import sections. Import builtin modules, then a blank line, then import any libraries, then a blank line, then local imports.

Reply:
I like doing that too, but I somehow forgot about it this time.